migration/20180112-redo-opds-for-non-author-contributors.sql

```diff
@@ -0,0 +1,14 @@
+-- Remove the 'generate-opds` coverage record for any works
+-- that have an associated series.
+delete from workcoveragerecords where operation='generate-opds' and work_id in (
+       select w.id from works w join editions e on w.presentation_edition_id=e.id where e.series is not null
+);
+
```

Comment:
This 'series' thing should have happened a long time ago but we didn't have any way of automatically regenerating the OPDS entries; now we do.